### PR TITLE
[dev-task] Add names within a pack as known words automatically in demsito-sdk doc-review

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/45735)

## Description
Added automatic ignore for the doc-reviewer. 
Now supports:
1. Packs name.
2. Integrations name.
3. Scripts name.
4. Integration commands names.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -827,6 +826,88 @@ def is_file_path_in_pack(file_path):
     return bool(re.findall(PACKS_DIR_REGEX, file_path))
 
 
+def add_default_pack_known_words(file_path):
+    """
+    Ignores the pack's content:
+    1. Pack's name.
+    2. Integrations name'.
+    3. Integrations command names'.
```
Comment (MosheEichler) [inline demisto_sdk/commands/common/tools.py:834]: Add the script name

Comment (abaumgarten) [reply demisto_sdk/commands/common/tools.py:834]: done

**Comment by abaumgarten** [comment]
@DeanArbel @DinaMeylakh @dorschw assigning all of you for code review, please feel free to comment / approve.

File: demisto_sdk/commands/common/tools.py [hunk 2]
```diff
@@ -827,6 +826,89 @@ def is_file_path_in_pack(file_path):
...
+    Returns: (set) of all the commands and integrations names found.
+
+    """
+    integrations_dir_path = os.path.join(PACKS_DIR, get_pack_name(file_path), INTEGRATIONS_DIR)
+    command_names: Set[str] = set()
+    if not glob.glob(integrations_dir_path):
+        return command_names
+
+    found_integrations: List[str] = os.listdir(integrations_dir_path)
+    if len(found_integrations) != 0:
```
Comment (dorschw) [inline demisto_sdk/commands/common/tools.py:869]:
```suggestion
    if found_integrations:
```

File: demisto_sdk/commands/common/tools.py [hunk 3]
```diff
@@ -827,6 +826,89 @@ def is_file_path_in_pack(file_path):
...
+
+    """
+    scripts_dir_path = os.path.join(PACKS_DIR, get_pack_name(file_path), SCRIPTS_DIR)
+    scripts_names: Set[str] = set()
+    if not glob.glob(scripts_dir_path):
+        click.secho(f'no scripts path found')
+        return scripts_names
+
+    found_scripts: List[str] = os.listdir(scripts_dir_path)
+    if len(found_scripts) == 0:
```
Comment (dorschw) [inline demisto_sdk/commands/common/tools.py:899]:
```suggestion
    if not found_scripts:
```

**Review by dorschw — APPROVED** [review]
Awesome job

File: demisto_sdk/commands/common/tools.py (hunk 2 above)
Comment (abaumgarten) [reply demisto_sdk/commands/common/tools.py]: done

File: demisto_sdk/commands/common/tools.py (hunk 3 above)
Comment (abaumgarten) [reply demisto_sdk/commands/common/tools.py]: Done

**Review by DeanArbel — APPROVED** [review]
Nice.

**Pull request merged**